packages/perseus/src/widgets/__tests__/plotter.test.tsx

```diff
@@ -14,14 +14,6 @@ describe("plotter widget", () => {
         setDependencies(testDependencies);
     });
 
-    it("should snapshot basic question", () => {
-        // Arrange and Act
-        const {container} = renderQuestion(question1);
-
-        // Assert
-        expect(container).toMatchSnapshot("initial render");
-    });
-
```

Comment:
This test didn't fail even when I threw an error in the plotter's render method. Tests that can't fail are useless, so I opted to delete this one.